<?xml version="1.0"?>
<launch>

  <node pkg="mbs_husky_utilities" type="pointcloud_filter_node" name="pointcloud_filter_node" output="screen" required="true">
    <param name="angle_min" value="-3.1415"/>
    <param name="angle_max" value="3.1415"/>
    <param name="c_radius"  value="0.70"/>
  </node>

</launch>
